src/zsales_cl_abapgit.clas.abap

```diff
@@ -67,7 +68,8 @@ CLASS ZSALES_CL_ABAPGIT IMPLEMENTATION.
              posnr,
              matnr,
              charg,
-             werks
+             werks,
+             zmeng                "SCR1 - 02.07.2020 - V1
```

Comment:
New field ZMENG added - as per FD - Ok
Please add Unit of Measure MEINS from VBAP - Review comments

Reply:
Unit of Measure added - Please Review